x-pack/test/common/services/security_solution/roles_users_utils.ts

```diff
@@ -5,32 +5,15 @@
  * 2.0.
  */
 
-import { assertUnreachable } from '@kbn/security-solution-plugin/common/utility_types';
-import {
-  t1AnalystUser,
-  t2AnalystUser,
-  hunterUser,
-  hunterNoActionsUser,
-  ruleAuthorUser,
-  socManagerUser,
-  platformEngineerUser,
-  detectionsAdminUser,
-  readerUser,
-  t1AnalystRole,
-  t2AnalystRole,
-  hunterRole,
-  hunterNoActionsRole,
-  ruleAuthorRole,
-  socManagerRole,
-  platformEngineerRole,
-  detectionsAdminRole,
-  readerRole,
-} from '@kbn/security-solution-plugin/server/lib/detection_engine/scripts/roles_users';
-
-import { ROLES } from '@kbn/security-solution-plugin/common/test';
+import serverlessRoleDefinitions from '@kbn/es/src/serverless_resources/security_roles.json';
+import essRoleDefinitions from '@kbn/security-solution-plugin/common/test/ess_roles.json';
+import { SecurityRoleName } from '@kbn/security-solution-plugin/common/test';
 import { FtrProviderContext } from '../../ftr_provider_context';
 
-export { ROLES };
+const allSupportedRoles = {
```

Comment:
Maybe create `allSupportedRoles` and export from `x-pack/plugins/security_solution/common/test/index.ts`

Reply:
I just didn't want to create one time use constant. Eventually decided to reexport `KNOWN_SERVERLESS_ROLE_DEFINITIONS` and `KNOWN_ESS_ROLE_DEFINITIONS` from `x-pack/plugins/security_solution/common/test/index.ts` as it helps to cover multiple scenarios.

I'd refrain from creating a common `ALL_ROLES` constants for now to make things more transparent.